.travis.yml

```diff
@@ -31,6 +35,9 @@ matrix:
         - os: osx
           language: generic
           env: TERRYFY_PYTHON='macpython 3.7.0'
+        - os: osx
+          language: generic
+          env: TERRYFY_PYTHON='macpython 3.8.0'
```

Comment:
Oh, is 3.8 available from terryfy already!  That's nice!

Reply:
[Apparently not](https://travis-ci.org/zopefoundation/zope.interface/jobs/610282558?utm_medium=notification&utm_source=github_status).

Reply:
I'm not a Mac user and know little about these things, but https://github.com/MacPython/terryfy looks nearly abandoned: last commit in 2018, no issues requesting Python 3.8 support, a README that directs users to https://github.com/matthew-brett/multibuild which "gets more development attention".

There's also https://github.com/joerick/cibuildwheel.

Reply:
OK, I know nothing about Mac builds.  How about I just drop this for now, and somebody with more clue can set it up?

Reply:
I've done that now.

Reply:
That was always my approach too: ignore 3.8 on Mac until some Mac user steps up. ;)